### PR TITLE
Fix release pages on docs

### LINKED ISSUE #3435: [Docs] Meshery release docs needs some update
#### Current State
For now some release versions are not present in the doc site

#### Desired State
The doc must be updated with relevant release pages

---

#### Contributor Resources

- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/meshery/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by alphaX86**
Signed-off-by: Aadhitya A <aadhitya864@gmail.com>

**Description**
This PR adds a new release page v0.5.31

This PR fixes #3435 

**Notes for Reviewers**
If any changes still needed, let me know :)

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 3726a002b25f811af6b6874ce37942b378638cb5 at: https://60e972fa5500d3de236c53d7--meshery-docs-preview.netlify.app

**Comment by alphaX86** [comment]
What is this new CI action?! OwO 

I thought Netlify is still present xD

File: docs/_releases/v0.5.31.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: release
+date: 2021-07-05
+tag: v0.5.31
+---
+
+## What's new
+**General**
+- Image pull policy updated @kumarabd (#3389)
```
Comment (adithyaakrishna) [inline docs/_releases/v0.5.31.md:9]: v0.5.31 Skipped by Release bot 😂

File: docs/_releases/v0.5.32.md [hunk 2]
```diff
@@ -28,9 +28,3 @@ tag: v0.5.32
 ## 📖 Documentation
 
 - [README] add: Community Forum @amino19 (#3272)
-
-## Contributors
-
-Thank you to our contributors for making this release possible:
-@Aisuko, @Chadha93, @Nikhil-Ladha, @Shreyas220, @amino19, @debjani1413, @dhruv0000, @hexxdump, @jeantiston, @killbotXD, @kumarabd, @l5io, @leecalcote, @metonymic-smokey, @navendu-pottekkat, @piyushsingariya, @samirparhi-dev, @suhaniii, @utkarsh-pro and l5io
-
```
Comment (adithyaakrishna) [inline docs/_releases/v0.5.32.md:36]: This update was necessary 💯 
Most of the releases before didn't have it but then some others did now every release is NEAT 🎉

**Review by adithyaakrishna — COMMENTED** [review]
Major Update and Cleaning 🌮 💯

File: docs/_releases/v0.5.32.md (hunk 2 above)
Comment (adithyaakrishna) [reply docs/_releases/v0.5.32.md:36]: @leecalcote Would love to know your opinion on this 👀

File: docs/_releases/v0.5.31.md (hunk 1 above)
Comment (alphaX86) [reply docs/_releases/v0.5.31.md:9]: OwO

**Comment by l5io** [comment]
🚀 Preview for commit be2d03d39882614e9d99f62d7d1589783bb2c681 at: https://60e97e1f7ad727d7f109874c--meshery-docs-preview.netlify.app

**Comment by alphaX86** [comment]
@adithyaakrishna emojis are removed now :)

File: docs/_releases/v0.5.15.md [hunk 3]
```diff
@@ -21,12 +21,12 @@ tag: v0.5.15
 - [UI] Navigation Menu Icon Scale with zoom in/out  @Ayush-Khandelwal-007 (#2966)
 - Bump nokogiri from 1.11.1 to 1.11.5 in /docs @dependabot (#2987)
 
-## 🧰 Maintenance
+## Maintenance
```
Comment (adithyaakrishna) [inline docs/_releases/v0.5.15.md:24]:
```suggestion
**Maintenance**
```

File: docs/_releases/v0.5.15.md [hunk 4]
```diff
@@ -21,12 +21,12 @@ tag: v0.5.15
...
 
 - [CI] Makefile: `make site` for conveniently building docs @leecalcote (#2993)
 - [CI] Helm: Test of helm chart releaser for remote repo @leecalcote (#2982)
 
-## 📖 Documentation
+## Documentation
```
Comment (adithyaakrishna) [inline docs/_releases/v0.5.15.md:29]:
```suggestion
**Documentation**
```

Comment (adithyaakrishna) [reply docs/_releases/v0.5.15.md:29]: Yooo @alphaX86  👋🏻 
Can you please change these across all the files, All the other stuff LGTM 💯🎉

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]

**Comment by alphaX86** [comment]
Let me do in local...

**Comment by adithyaakrishna** [comment]
UKW 😂 
My Mail is aadithya794@gmail.com
Yours's is aadhitya864@gmail.com

The similarity 😂💯

**Comment by alphaX86** [comment]
> UKW 😂
> My Mail is [aadithya794@gmail.com](mailto:aadithya794@gmail.com)
> Yours's is [aadhitya864@gmail.com](mailto:aadhitya864@gmail.com)
> 
> The similarity 😂💯

I thought I was the one who's unique LOL power of numbers

**Comment by l5io** [comment]
🚀 Preview for commit e6332a615b5a8f3cde1ee8b758e3dfa93e23e575 at: https://60e98435af97d6eff07462f0--meshery-docs-preview.netlify.app

**Review by adithyaakrishna — DISMISSED** [review]
Looks good to me 🎉🎊
Awesome work! @alphaX86 🎈

File: docs/_releases/v0.5.16.md [hunk 5]
```diff
@@ -12,13 +12,13 @@ tag: v0.5.16
 - [update] Filter UI @Chadha93 (#3005)
 - Add platform support to bash script @navendu-pottekkat (#2997)
 
-## Features
+**Features**
 - [mesheryctl] Improved UX for mesheryctl system check --pre @piyushsingariya (#3027)
 
-## 🧰 Maintenance
```
Comment (leecalcote) [inline docs/_releases/v0.5.16.md:18]: Gents, a verbatim copy of the release notes to a new .yaml file with appropriate frontmatter in the Docs is all that is needed. No reformatting needs to be done.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/_releases/v0.5.16.md (hunk 5 above)
Comment (alphaX86) [reply docs/_releases/v0.5.16.md:18]: Copy that, but what about the ones that are released? So far I made changes only to ones that are released and published in docs. For future ones... IG we can fix the CI action and take care of that

Comment (alphaX86) [reply docs/_releases/v0.5.16.md:18]: Well I'll reconsider then....

Comment (adithyaakrishna) [reply docs/_releases/v0.5.16.md:18]: Okay, affirmative captain 👀

**Comment by l5io** [comment]
🚀 Preview for commit ecc96ecc6a8b30af8847cd2a1bff6226876ed288 at: https://60ea8f0a41189b3ff252581b--meshery-docs-preview.netlify.app

**Comment by alphaX86** [comment]
@leecalcote @adithyaakrishna seems a major error popped up while rebasing the branch as it included every file so I'm deleting this PR and the branch and make a new one with no reformatting

**Comment by adithyaakrishna** [comment]
Gothcha @alphaX86 💯

**Pull request closed without merging**